wagtailnhsukfrontend/templates/wagtailnhsukfrontend/panels.html

```diff
@@ -0,0 +1,6 @@
+<div class="nhsuk-panel{{ class }}">
```

Comment:
The logic around creating this class is split over the template/block code, which works but is not very readable.

It would be better to _either_ do it all in the `block.py` _or_ all in the template.

All in the block:
```python
        context['class'] = {
            'plain': 'nhsuk-panel',
            'labeled': 'nhsuk-panel-with-label',
            'grey': 'nhsuk-panel nhsuk-panel--grey',
        }[value['type']]
```

All in the template:
```
<div class="nhsuk-panel{% if value.type == "labeled" %}-with-label{% elif value.type == "grey" %} nhsuk-panel--grey{% endif %}">
```

Reply:
**however**
It is probably better to have an optional title field and an "is grey" checkbox instead of a 3-choice dropdown.

If the title has a value, output the h3 and the `-with-label` class.
If "is grey" has been ticked, output the class to make it grey.

Reply:
I had a think about it and I agree that it is probably better to do it that way